src/Resources/views/CRUD/base_edit_form.html.twig

```diff
@@ -8,16 +8,11 @@
             {{ "form_not_available"|trans({}, "SonataAdminBundle") }}
         </div>
     {% else %}
-        <form
-              {% if sonata_config.getOption('form_type') == 'horizontal' %}class="form-horizontal"{% endif %}
-              role="form"
-              action="{% block sonata_form_action_url %}{{ admin.generateUrl(url, {(admin.idParameter): objectId, 'uniqid': admin.uniqid, 'subclass': app.request.get('subclass')}) }}{% endblock %}"
-              {% if form.vars.multipart %} enctype="multipart/form-data"{% endif %}
```

Comment:
this is missing

Reply:
This should automatically be rendered by the `form_start` function